### PR TITLE
#1452 changed dynamic groupId in master-pom.xml to fixed

### PR DESCRIPTION
**Pull request by jan-vcapgemini**
separated all templates sets into deployables
moved util tests into templates-devon4j-utils plug-in
renamed devon4j-templates to devon4j-templates-tests

Adresses #1452.

@devonfw/cobigen

### CODE REVIEW

**Review by LarsReinken — APPROVED** [review]

File: cobigen-templates/pom.xml [hunk 1]
```diff
@@ -27,12 +50,9 @@
 
   <build>
     <resources>
-      <resource>
-        <directory>src/main/templates</directory>
-        <targetPath>src/main/templates</targetPath>
-      </resource>
```
Comment (MikeSchumacherCapgemini) [inline cobigen-templates/pom.xml:33]: Taking this out doesn't hurt backwards compatibility?

File: cobigen-templates/security-permissions/src/main/resources/context.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (MikeSchumacherCapgemini) [inline cobigen-templates/security-permissions/src/main/resources/context.xml:1]: Is this supposed to lie in /src/main/resources?

**Review by MikeSchumacherCapgemini — COMMENTED** [review]
Have you checked/thought about overall backwards compatibility?

File: cobigen-templates/pom.xml (hunk 1 above)
Comment (jan-vcapgemini) [reply cobigen-templates/pom.xml:33]: It's still there, just moved to line 52.
I've just changed our IDE path to src/main/resources.
The path in the jar remains src/main/templates.

**Pull request merged**